bukuserver/middleware/flask_reverse_proxy_fix.py

```diff
@@ -0,0 +1,60 @@
+# clone of flask-reverse-proxy-fix with github fixes from @rachmadaniHaryono and @Glocktober
```

Comment:
Namely [this](https://github.com/antarctica/flask-reverse-proxy-fix/pull/1) and [this](https://github.com/antarctica/flask-reverse-proxy-fix/pull/2) fixes